builder/steps/gen-dockerfile/contents/data/Dockerfile.txt

```diff
@@ -38,6 +40,11 @@ COPY --from=build_step /app .
 # single application.
 FROM <%- baseImage %>
 COPY . /app/
+<% if (config.nodeVersion) { %>
```

Comment:
This is inside the else statement started at line 38, right? Indentation can help the readability, and it seems to be doing so in this file. e.g. line 68 of this commit.